lib/manager/components/FeedSourceTableRow.js

```diff
@@ -338,13 +388,9 @@ class FeedActionsDropdown extends PureComponent<Props> {
     }
   }
 
-  render () {
-    const {
-      feedSource,
-      project,
-      user
-    } = this.props
-
+  // This method is extracted variable generation which had to be split
```

Comment:
Sentence is incomplete. Also use `/**`-style.

Reply:
Fixed!